Tests/LeafTests/LeafTests.swift

```diff
@@ -54,6 +54,44 @@ class LeafTests: XCTestCase {
             XCTAssertEqual(res.body.string, "Hello vapor @ /test-file")
         }
     }
+    
+    func testContextUserInfo() throws {
+        var test = TestFiles()
+        test.files["/foo.leaf"] = """
+        Hello #custom()!
+        """
+
+        struct CustomTag: LeafTag {
+            
+            func render(_ ctx: LeafContext) throws -> LeafData {
+                let info = ctx.userInfo["info"] as? String ?? ""
```

Comment:
Since you already have access to `ctx.application.storage` here, do we also need `userInfo`?

Reply:
The main idea was to use the userInfo property as a place to store environment variables for custom Leaf tags. Should I use the storage property for this purpose instead? I thought that's only for storing application related stuff, so I don't wanted to save Leaf data there. Am I missing something?

Reply:
`app.storage` _could_ be used to do this but I can see how `app.leaf.userInfo` is easier and more straight forward to use.